### DISCUSSION TITLE
Reduce value of chi^2 in cigale

### DISCUSSION THREAD

**Discussion by Mayuri262** [mboquien/cigale#95] [Q&A]
How do I reduce the value of chi^2 to get the best fit in cigale? I am an undergraduate student and completely new to cigale so I am not getting which parameters to change and based on what.

Can someone please guide me on how can I reduce chi^2 value.

**Comment by mboquien** [comment]
As I mentioned in your other questions, you may want to see whether the model grid you have selected is appropriate. You can try to expand the number of models, expanding the range that is currently probed.

**Comment by Mayuri262** [comment]
Thank you for your reply.
 Actually as far as my internship is concerned I am working on one source
only. So I found out datas on various wavelengths for that source and made
the table as required by cigale to work on and tried running the program.
I got the plot of one source but the chi² value is too large. I want to
reduce it but I am not exactly sure how to do it.

**Reply by mboquien** [reply]
As I said, you may want to increase the number of models. The default `pcigale.ini` configuration only builds a single model so it can run. In practice one wants thousands to millions of models. Simply add more values to the relevant parameters.

**Comment by Mayuri262** [comment]
When you said "add more values to the relevant parameters" what exactly do
you mean?

**Reply by mboquien** [reply]
It depends on your problem. For instance you may have a dust emission module with a parameter called `alpha`. By default `pcigale.ini` only one value, `2.0`. Then you may want to add other values, with the possible values as indicated in the comments for the parameter in the `pcigale.ini` file. The values must be relevant to the object you are studying. This is something you probably want to discuss with your advisor.

**Reply by shivathanay-p** [reply]
When we add multiple values in the `pcigale.ini,` does CIGALE build models for all the values provided, and does CIGALE take the best one? Or does a parameter only allow one value, and we have to change the value of the parameter each time we run CIGALE?

**Reply by mboquien** [reply]
It builds a Cartesian grid with all the possible combinations of the parameters. Parameters can have one or more values. CIGALE compute the exact best-fit and also provides likelihood-weighted estimates.

**Reply by shivathanay-p** [reply]
I have seen possible values for each parameter in the `pcigale.ini` file. Wouldn't it be better to add every possible value, then? And build all the models possible to get the least chi^2 value possible, or would that be too many models to compute?

**Reply by mboquien** [reply]
The grid would be too big and some regions in the parameter space may not be appropriate for the modeled galaxies.

**Comment by Mayuri262** [comment]
Thank you for the clarification.

Is there any paper you can advice me to read so as to get an idea of what
all parameters I have to change in the pcigale.ini configuration file and
what significance the parameters has .

**Reply by mboquien** [reply]
The [article presenting CIGALE](https://ui.adsabs.harvard.edu/abs/2019A%26A...622A.103B/abstract) would be a good starting point I think.

**Comment by Mayuri262** [comment]
Thanks, I'll check it out.
